src/main/resources/application.properties

```diff
@@ -8,8 +8,9 @@ spring.jpa.hibernate.ddl-auto=validate
 spring.jpa.show-sql=true
 spring.jpa.open-in-view=false
 
-server.servlet.context-path=/api
+telegram.bot.username=${BOT_NAME}
+telegram.bot.token=${BOT_TOKEN}
+telegram.bot.group.id=${GROUP_ID}
 
 jwt.expiration=${EXPIRATION_TIME}
 jwt.secret=${SECRET_KEY}
```

Comment:
It is preferred to call them clearly.
jwt.expiration=${JWT_EXPIRATION}
jwt.secret=${JWT_SECRET}